frontend/src/components/VendorInfoNav.css

```diff
@@ -53,6 +53,5 @@
   font-family: 'Amatic SC', sans-serif;
   font-size: 32px;
   font-weight: 700;
-  margin: 25px 0;
-  padding-left: 20px;
+  padding-left: 15px;
```

Comment:
A 20px jó volt, legyen annyi oldalt a padding.

Reply:
A többi szekciónál is rossz, ahogy nézem. Akkor maradjon 15px ebben a PR-ben és majd veszek föl másik issue-t.

Reply:
Itt a külön issue, erre legyen külön branch meg PR majd: https://github.com/flowacademyhu/piac-app/issues/179

Reply:
Oké, köszi szépen!